Firmware/HomeButtonsArduino/src/mqtt_helper.cpp

```diff
@@ -252,7 +253,8 @@ void MQTTHelper::update_discovery_config() {
         FormatterType{} + _device_state.factory().unique_id + "_temperature";
     temp_conf["stat_t"] = t_temperature();
     temp_conf["dev_cla"] = "temperature";
-    temp_conf["unit_of_meas"] = "°C";
+    temp_conf["unit_of_meas"] =
+        _device_state.get_use_fahrenheit() ? "°F" : "°C";
```

Comment:
Same remark I already did, I believe MQTT should only send temperature in Celsius.